File: c/swap.c
// Swapping of two numbers
#include<stdio.h>
int main()
{
	int a,b,temp;
	printf("Enter a and b values \n");
	scanf("%d %d",&a,&b);
	printf("The values before swapping is a= %d and b= %d \n",a,b);
	temp=a;
	a=b;
	b=temp;
	printf("The values after swapping is a= %d and b= %d \n",a,b);
}

File: docs/java/swap.java
// Swapping of two numbers
import java.io.*;
class swap
{
	public static void main(String args[])throws IOException
	{
		int a,b,temp;
		DataInputStream dis=new DataInputStream(System.in);
		System.out.println("Enter the values of a and b");
		a=Integer.parseInt(dis.readLine());
		b=Integer.parseInt(dis.readLine());
		System.out.println("The values before swapping is a= "+a+" and b= "+b);
		temp=a;
		a=b;
		b=temp;
		System.out.println("The values after swapping is a= "+a+" and b= "+b);
	}
}